enforcer/enforcer_test.go

```diff
@@ -0,0 +1,251 @@
+// Copyright 2021 The Rode Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package enforcer
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+
+	"github.com/golang/mock/gomock"
+	"github.com/google/go-containerregistry/pkg/name"
+	registryv1 "github.com/google/go-containerregistry/pkg/v1"
+	registryfake "github.com/google/go-containerregistry/pkg/v1/fake"
```

Comment:
this is super helpful

Reply:
For sure. It's generated by [counterfeiter](https://github.com/maxbrunsfeld/counterfeiter), which seems like a nice alternative to gomock.